### DISCUSSION TITLE
Import your own voice model?

### DISCUSSION THREAD

**Discussion by Pri0ri** [PaulWoitaschek/Voice#1347] [Ideas]
Just a thought have you considered adding an ability to map one's own voice to create a voice model. I figured it might be fairly tedious basically reading a dictionary unless you do something phonetically but I thought it might be cool

**Comment by ahmuro** [comment]
I understand what you mean, I just don't see how would this relate to Voice. This is basically an audio file player, we already have the audio recorded, and this app just plays it as it was recorded. If you are talking about TTS, maybe your needs would be better served by a PDF or ebook reader that has TTS functionality. 

Edit to add something maybe you would find interesting https://www.descript.com/overdub
